components/inputs/input-date-time-range.js

```diff
@@ -85,25 +104,39 @@ class InputDateTimeRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 	}
 
 	render() {
-		const startLabel = this.startLabel ? this.startLabel : this.localize('components.input-date-time-range.startDate');
-		const endLabel = this.endLabel ? this.endLabel : this.localize('components.input-date-time-range.endDate');
+		const tooltipStart = (this.validationError && !this._startDropdownOpened) ? html`<d2l-tooltip align="start" for="${this._startInputId}" state="error">${this.validationError}</d2l-tooltip>` : null;
```

Comment:
One issue with having the tooltips attached to the `d2l-input-date-time` components is that if the tooltip opens above it'll be placed above the label rather than above the input box. A fix would be to set `position="bottom"` so  it always opens below but that would cause the tooltip to position itself slightly less optimally.
![Capture](https://user-images.githubusercontent.com/11379933/89813559-ddc6b100-db0f-11ea-8e15-98644c9e8f01.PNG)